couler/argo_submitter.py

```diff
@@ -160,3 +165,15 @@ def _create_secret(self, secret_yaml):
         return self._core_api_client.create_namespaced_secret(  # noqa: E501
             self.namespace, secret_yaml
         )
+
+    @classmethod
+    def set_default(cls, submitter=None):
```

Comment:
Where is this being used?

Reply:
I haven't supplied code that is using it, I'll add some to the examples.

Reply:
see #213 , I've accidentally added that code here.